resources/views/livewire/top-ten-leaderboard.blade.php

```diff
@@ -16,12 +16,15 @@
                 <thead>
                     <tr>
                         <th>Place</th>
-                        @if ($analyticClass->type()->isNot(AnalyticType::ONLY_GAME()))
+                        @if ($analyticClass->type()->notIn([AnalyticType::ONLY_GAME(), AnalyticType::MAP()]))
                             <th>Gamertag</th>
                         @endif
                         @if ($analyticClass->type()->isGame())
                             <th>Game</th>
                         @endif
+                        @if ($analyticClass->type()->is(AnalyticType::MAP()))
```

Comment:
Since we've done `->is(AnalyticType::MAP()` twice in this view, we could probably make a new helper on the Analytic class for `isMap()`